README.md

```diff
@@ -45,9 +45,9 @@ Or add the `HarmonyColorPicker` to your Compose hierarchy for an HSV color wheel
  HarmonyColorPicker(
     harmonyMode = harmonyMode.value,
     modifier = Modifier.size(400.dp),
-    onColorChanged = { hsvColor ->
-        currentColor.value = hsvColor.toColor()
-        extraColors.value = hsvColor.getColors(colorHarmonyMode = harmonyMode.value)
+    onValueChanged = { color ->
```

Comment:
﻿﻿We must keep both `HarmonyColorPicker` and `HarmonyColorPicker` the same naming, `onValueChange` or `onColorChange`. I don't have a strong opinion about which one to choose.

Reply:
I'm a little unclear, are you talking about the two HarmonyColorPicker functions or did you mean ClassicColorPickerScreen and HarmonyColorPicker needed to use the same naming?

If it's the two versions of HarmonyColorPicker I'll note the reason I changed from `onColorChanged` here to `onValueChanged` is because the passed in field name changed from `color` to `value` to keep the signatures for the `Color` old version and the new version that takes in `HsvColor` instead.   I will note that the version with `onColorChanged` is deprecated with a replace with to make it easy to migrate to the new version for upgraders.

In case the issue is the inconsistency between classic and harmony I'm making a push that changes them to be consistent in the non-deprecated version

Reply:
On second thought I just updated it to use the old naming for both.  I think the better distinction between deprecated and non-deprecated is useful because it can help prevent someone from using the old one by mistake.  But I don't feel anywhere near strongly enough to leave it that way

Reply:
in this PR the non deprecated Composables are defined in this way
```kotlin
@Composable
fun ClassicColorPicker(
    modifier: Modifier = Modifier,
    hsvColor: HsvColor = HsvColor.from(Color.Red),
    showAlphaBar: Boolean = true,
    onColorChanged: (HsvColor) -> Unit
) {

@Composable
fun HarmonyColorPicker(
    modifier: Modifier = Modifier,
    harmonyMode: ColorHarmonyMode,
    value: HsvColor = HsvColor.from(Color.Red),
    onValueChanged: (HsvColor) -> Unit
) {
   
```
We should keep the parameter names in sync. 

> On second thought I just updated it to use the old naming for both. I think the better distinction between deprecated and non-deprecated is useful because it can help prevent someone from using the old one by mistake. But I don't feel anywhere near strongly enough to leave it that way

Agree with that please update ClassicColorPicker to match HarmonyColorPicker with value/onValueChange.